tpm2_cmds.c

```diff
@@ -35,6 +35,9 @@
 static int tpm2_alloc_cmd(struct tpmbuff *b, struct tpm2_cmd *c, u16 tag,
 		u32 code)
 {
+	/* ensure buffer is free for use */
+	tpmb_free(b);
```

Comment:
Should this be done in the tpm1_cmd.c code too?

Reply:
Actually, this along with the next couple of lines are a better pattern. Will use it in tpm1_cmd.c